src/main/java/frc/robot/RobotContainer.java

```diff
@@ -227,8 +223,6 @@ private void configureButtonBindings() {
                 .whileTrue(intake.outtake().alongWith(gripper.setRollerPower(-0.7)))
                 .onFalse(intake.stop().alongWith(gripper.setRollerPower(0)));
 
-        xboxController.start().onTrue(elevator.lock());
-        xboxController.back().onTrue(elevator.unlock());
```

Comment:
Isn't there a mechanical stop on the climber anymore? Won't the robot fall down too fast?

Reply:
12b0df6d69420930a2e37a642c7d4d4680acb9db